### ISSUE TITLE
Terminal character spacing too wide to be usable

### ISSUE THREAD

**Issue by alecstein** [lapce/lapce#2784]
## Lapce Version

0.3.0 (1)

## System information

macOS Monterey 12.6.3

## Describe the bug
Terminal font is borked. Character spacing is too wide.

## Additional information

This was immediately after downloading and opening Lapce.

<img width="974" alt="image" src="https://github.com/lapce/lapce/assets/16236421/55384fec-c90e-4e4c-954f-7926fde14e4b">

**Comment by alecstein** [comment]
<img width="974" alt="image" src="https://github.com/lapce/lapce/assets/16236421/e9a49df0-d988-4934-9406-9b644bbd058a">
Just tried that. I used Apple+, to open the settings pane and tried to set the font to `Monospace`, but opening a new terminal window didn't do anything. I wasn't sure if I was supposed to "save" the settings somehow, but I couldn't figure out how. The settings pane itself actually seems a little bit glitchy on my machine, with some parts at the bottom being cut off.

**Comment by dzhou121** [comment]
This issue is fixed in nightly where the default font is set to monospace and a monospace font is correctly used.

**Comment by RustoMCSpit** [comment]
does this need to be closed?

**Comment by lalomartins** [comment]
I don't think so. Setting it to `monospace` kind of works (the text is tiny and not Unicode so not really, but it's barely usable); but manually setting it to an installed monospace font goes back to the state shown in the screenshot.
